src/BgLZ-3-0/Cert/_reference/resources/bglz-0-1-AllergieIntolerantie-01.xml

```diff
@@ -1,3 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure if this is a problem? It is sometimes added.. is it added by the script?

Reply:
I've missed this addition that has been automatically added by the script, so I'll delete this.